site/public/css/diff-viewer.css

```
@@ -1,10 +1,14 @@
 @import url("google/inconsolata.css");
 
-/*.diff-container {*/
-/*    background-color: var(--standard-pale-gray);*/
-/*    border: 1px solid var(--standard-light-gray);*/
-/*    width: 100%;*/
-/*}*/
+/* .diff-container { */
+
+/*    background-color: var(--standard-pale-gray); */
+
+/*    border: 1px solid var(--standard-light-gray); */
+
+/*    width: 100%; */
+
+/* } */
```

Spacing this out over multiple lines is weird, probably better to have it be a singular block comment:
```suggestion
/*
.diff-container {
    background-color: var(--standard-pale-gray);
    border: 1px solid var(--standard-light-gray);
    width: 100%;
}
*/
```

Though perhaps just get rid of it since it's been commented out a year, and there's no real explanation on it.

---

Ok, I'll get rid of it then.